metrics-operator/api/v1alpha3/keptnmetric_types.go

```diff
@@ -60,6 +60,9 @@ type RangeSpec struct {
 	Interval string `json:"interval,omitempty"`
 	// Step represents the query resolution step width for the data query
 	Step string `json:"step,omitempty"`
+	// Aggregation defines as the type of aggregation function to be applied on the data
```

Comment:
Please also add the allowed values to the description of the property, so they are also visible in the docs

Reply:
Fixed